### DISCUSSION TITLE
Naming: page listing ranking of email clients by feature category

### DISCUSSION THREAD

**Discussion by husseinalhammad** [email-markup-consortium/email-markup-consortium#85] [Q&A]
## Background
For context see:
- [2022-08-09 meeting notes](https://github.com/email-markup-consortium/email-markup-consortium/blob/main/meeting-notes/2022-08-09.md#email-clients-ranking-by-feature-category) 
- https://github.com/email-markup-consortium/email-markup-consortium/discussions/40#discussioncomment-2202448
- https://github.com/hteumeuleu/caniemail/issues/232

In short, this is a page listing the number of features supported by email clients in various categories. This visualises data coming from [caniemail.com](https://caniemail.com), which we helped populate.

- Email Client X
  - accessibility 3/10
  - performance 4/4
- Email Client Y
  - accessibility 6/10
  - ..... 

## Draft page
We have a draft PR in the website repo. Deploy preview is available.
https://github.com/email-markup-consortium/emailmarkup.org/pull/24

## Names

We are yet to settle on a name for this page and currently looking for the community and members input. The current working name is "email clients feature ranks".  

- Is this a good descriptive name?
- Do you have an idea for a more fitting name?

Note that the meeting notes include ideas on how this can be used in the future. These ideas may or may not affect your opinions when it comes to naming this page.

**Comment by dylanatsmith** [comment]
> email clients feature ranks

I feel like this could be confused for actual features of the email clients (ex: snoozing messages, scheduled sends, warning about missing attachments). Maybe adding "support" would help clarify. Something like `Email client feature support rankings`.

**Reply by husseinalhammad** [reply]
I agree with you on this!  Do we dare to make it even more verbose?

> Email client HTML/CSS feature support rankings
